git/README-es.md

```diff
@@ -43,25 +45,51 @@ A continuación, listaremos el contenido de este documento. Algunos de los linea
 En esta sección se explica el manejo de cada una de las branches involucradas en el proceso de desarrollo previo a salir a producción.
 
 
-### Develop branch (develop)  **[Required]**
+### Develop branch (develop)  **[Requerida]**
 
 Es la branch principal, refleja la integración de todo el trabajo del equipo durante la etapa de desarrollo del proyecto.
 Es la branch base hasta salir a producción por primera vez.
 
 * **Origen:** *master* (cuando se inicia el proyecto, es creada desde master)
 * **Destino:** -
-* **Casos de uso:** Integración del trabajo del equipo
+* **Casos de uso:** Integración del trabajo del equipo.
 
 
+### Staging branch (staging) **[Requerida]**
+
+Esta branch contiene todas las funcionalidades listas para ser testeadas por el Cliente.
+
+Si el proyecto tiene CD configurado, cada vez que algo sea mergeado en esta branch el código será automáticamente deployado al ambiente de Staging para que el Cliente pueda probarlo.
+
+* **Origen:** *master* (cuando se inicia el proyecto, es creada desde master)
+* **Destino:** -
+* **Casos de uso:** Integración del trabajo hecho por el equipo de desarrollo ya testeado y aprobado por el equipo de QA, listo para ser revisado por el Cliente.
+
+### QA branch (qa) **[Opcional]**
+
+Esta branch contiene todas las funcionalidades listas para ser testeadas por el equipo de QA.
+
+Si el proyecto tiene CD configurado, cada vez que algo sea mergeado en esta branch el código será automáticamente deployado al ambiente de QA.
+
+* **Origen:** *master* (cuando se inicia el proyecto, es creada desde master)
```

Comment:
This should be `**Origen:** *develop*` right? 🤔

Reply:
You create this branch once from master, and then you merge things from develop, but the origin of the branch is not develop.

Reply:
Probably someone could be tempted to crate `QA` from `develop` once that branch has something new for the first time to be tested. But, make total sense to create `QA` and `develop` from the beginning using master as base always.